scss/_list.scss

```
@@ -0,0 +1,45 @@
+// Юля
+.list {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+
+  &__img {
+    width: 140px;
+    height: 140px;
+    border-radius: 50%;
+    background: linear-gradient(51deg, #3595ff 2.94%, #3564fc 140.18%);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 54px;
+    position: relative;
+    left: 50%;
+    transform: translateX(-50%);
+
+    &:hover {
+      background: none;
+      background-color: #fff;
+      border: 1px solid #3595ff;
+    }
+
+    &:hover .img__svg {
+      fill: #3564fc;
+    }
```

ховери :+1: 
можна ще було скоротити
```suggestion
    &:hover {
      background: #fff;
      border: 1px solid #3595ff;      
      .img__svg {
          fill: #3564fc;
      }
    }
```